### DISCUSSION TITLE
make launch failed!

### DISCUSSION THREAD

**Discussion by AndyBestaa** [carla-simulator/carla#8245] [Q&A]
![屏幕截图 2024-10-07 160854](https://github.com/user-attachments/assets/6e5744b9-100b-442b-a54a-ae67fd61d45a)

All the previous steps have been completed. At the last step, every time CarlaUE4 reaches 95%, a window will pop up.
![屏幕截图 2024-10-07 162022](https://github.com/user-attachments/assets/3149976b-dd20-4cdd-819b-2ea36a8d691d)

![屏幕截图 2024-10-07 160812](https://github.com/user-attachments/assets/c56c9e9b-3cf1-4058-834c-39c6bc15ac72)
What are the possible reasons for this? How should it be resolved?

**Comment by Jinviz** [comment]
Which do you use version of WinSDK?

**Reply by afriyantiliza** [reply]
Hi, I have the same problem
I use WinSDK=10 or 11

**Comment by Blyron** [comment]
Please, provide logs in english.
